graphdatascience/query_runner/arrow_version.py

```diff
@@ -0,0 +1,17 @@
+from enum import Enum
+
+
+class ArrowVersion(Enum):
```

Comment:
Nitpick: I think the `ArrowVersion` naming here can be confused with the actual version of the Arrow library (on client or server). Would it be better to call it `ArrowEndpointVersion` or even `ServerArrowEndpointVersion`, or something else that disambiguates it

Reply:
Good point. I need to change the version lookup anyways, so I can also make that rename. `ArrowEndpointVersion` sounds good to me.